### ISSUE TITLE
osx version trouble for conda install?

### ISSUE THREAD

**Issue by mmotoc** [nextgenusfs/funannotate#813]
I am trying to install funannotate through conda using the following: 

conda install -c bioconda funannotate

And I am getting this error:

The following specifications were found to be incompatible with your system:

  - feature:/osx-64::__osx==10.16=0
  - feature:|@/osx-64::__osx==10.16=0
  - funannotate -> matplotlib-base -> __osx[version='>=10.12']

What can I do to fix this?

Thank you!

**Comment by hyphaltip** [comment]
I'm trying to see if I can get lighttpd installed on osx conda forge which is stopping me from testing on osx here.

**Comment by hyphaltip** [comment]
@nextgenusfs have you been able to install things via conda okay on OSX?

**Comment by nextgenusfs** [comment]
Several tools were disabled because they weren't working several years ago on OSX. Probably need to manually install augustus, PASA, Trinity, etc: https://github.com/bioconda/bioconda-recipes/blob/master/recipes/funannotate/meta.yaml

I'm not entirely sure if `conda create -n new_env funannotate` will work on OS X, effectively I'm doing it somewhat manually on my Mac, I'll update here if I get something that works.

**Comment by nextgenusfs** [comment]
Update: currently still unable to get a version of augustus that works with the --proteinprofile mode.....

**Comment by hyphaltip** [comment]
yeah it won't for me right now - I think we could make an osx package for the tools which don't conflict and then provide instructions for pasa, augustus installs through homebrew etc.

**Comment by MichaelTiemannOSC** [comment]
I came here having the same problem trying to install a different set of packages.  Alas, no answer yet to be found...
